src/resources/authoring/gv2.json

```diff
@@ -379,6 +379,104 @@
       },
       "visibleGenes": "wings, forelimbs, hindlimbs, armor, horns"
     },
+    "clutch-5drakes-starterTraits": {
+      "container": "FVContainer",
+      "template": "ClutchGame",
+      "challengeType": "create-unique",
+      "interactionType": "select-chromosomes",
+      "instructions": "Create 5 different baby drakes!",
+      "showUserDrake": true,
+      "randomizeTrials": true,
+      "changeableDrakes": [
```

Comment:
I used `changeableDrakes` and `staticDrakes` here so that each trial could be used in multiple ways - either for a challenge where mother, father, or all genes can be manipulated